### DISCUSSION TITLE
How to set the power_values values in hardware json file?

### DISCUSSION THREAD

**Discussion by kigik** [ExpressLRS/ExpressLRS#2902] [Q&A]
What does the power_values list in hardware json mean,  TX dBm?   is there any document to ref?

**Comment by pkendall64** [comment]
There is no reference for this. The values depend on the hardware. Some hardware uses a DAC to control the power amplification and the values would be DAC values. Some use different power amplifiers with different gains and so the power values will set the output from the semtech radio chip to be different to reach the desired final output.
